src/app/screens/Dashboard/screens/TodoList/components/Todos/components/Todo/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import { TodoI } from '~utils/types';
+import CustomCheckbox from '~components/Checkbox';
+import { useDispatch } from '~contexts/TodosContext';
+import { actionCreators } from '~contexts/TodosContext/reducer';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  todo: TodoI;
+}
+
+function Todo({ todo }: Props) {
```

Comment:
I'd name this `TodoItem` and the interface `Todo` o `TodoTask`, what do you think?